src/main/java/org/icgc/argo/workflow_management/service/model/Metadata.java

```diff
@@ -0,0 +1,15 @@
+package org.icgc.argo.workflow_management.service.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+
+@Getter
+@Builder
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Metadata {
```

Comment:
again, these models for nextflow error events when no weblog are extremely nextflow specific and should be explicitly named that way

Reply:
could even replace all the new models with the existing Nextflow Weblog classes???